projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -530,6 +531,11 @@ public static function gutenblock_render_field_consent( $atts, $content ) {
 	 * @return string HTML for the file upload field.
 	 */
 	public static function gutenblock_render_field_file( $atts, $content ) {
+		if ( apply_filters( 'jetpack_unauth_file_upload_plan_check', true ) && ! Plan::supports( 'field-file' ) ) {
```

Comment:
Is this filter just to help with testing?

Reply:
Testing and development. Setting it to `false` will allow you to add it without having to get the upgrade.

Reply:
We can consider removing the filter entirely once we're done with the development? It's comfy though